app/src/main/java/comcmput301f16t01/github/carrier/RequestAdapter.java

```diff
@@ -66,28 +67,28 @@ public View getView(int position, View v, @NonNull ViewGroup parent) {
             if (statusImageView != null) {
                 switch( request.getStatus() ) {
                     case( Request.OPEN ):
-                        //statusImageView.setImageResource( *** enter image for this status? *** );
-                        statusImageView.setBackgroundResource(R.color.openStatus);
+                        statusImageView.setImageResource(R.drawable.open);
+                        //statusImageView.setBackgroundResource(R.color.openStatus);
```

Comment:
There's a lot of commented out lines, which is totally fine. I probably do this lots. It just kinda makes the code a bit messy.